lib/fastlane/plugin/wpmreleasetoolkit/helper/github_helper.rb

```diff
@@ -75,11 +75,15 @@ def get_last_milestone(repository)
       #
       # @param [String] repository The repository name, including the organization (e.g. `wordpress-mobile/wordpress-ios`)
       # @param [String] title The name of the milestone we want to create (e.g.: `16.9`)
-      # @param [Time] due_date milestone due date (e.g. `2022-10-22T12:00:00Z`)
-      # @param [Integer] days_until_submission Number of days until submission
+      # @param [Time] due_date Milestone due date—which will also correspond to the code freeze date
+      # @param [Integer] days_until_submission Number of days from code freeze to submission to the App Store / Play Store
       # @param [Integer] days_until_release Number of days from code freeze to release
       #
       def create_milestone(repository:, title:, due_date:, days_until_submission:, days_until_release:)
+        UI.user_error!('days_until_release must be greater than zero.') if days_until_release <= 0
+        UI.user_error!('days_until_submission must be greater than zero.') if days_until_submission <= 0
+        UI.user_error!('days_until_release must be greather than days_until_submission') if days_until_submission >= days_until_release
```

Comment:
It feels to me that this would be more readable if the conditions matched the messages described in the `user_error!`, rather than being the negative of those ("must be greater than zero" --> use a `days_until-release > 0` condition), otherwise it needs your brain to think what the negative should be to ensure that the `if` is correct and matching the message.

In fact, for the 3rd case I think the condition and messages are incorrect, because it could be valid to consider that the submission and the release could be on the same day (that's not a common/used setup for any of our apps currently, but might be one day, at least it shouldn't be invalid imho)

```suggestion
        UI.user_error!('days_until_release must be greater than zero.') unless days_until_release > 0
        UI.user_error!('days_until_submission must be greater than zero.') unless days_until_submission > 0
        UI.user_error!('days_until_release must be greater or equal to days_until_submission') unless days_until_release >= days_until_submission
```

Reply:
Thanks for the feedback, I was not aware of the `unless` keyword, indeed is more legible that way. 😄